### PR TITLE
Add strict and warnings to the test suite

### PR DESCRIPTION
**Pull request by atoomic**

### CODE REVIEW

**Comment by atoomic** [comment]
note that #6 is also part of this PR to confirm that this does not introduce a regression,
you can view the build here https://github.com/atoomic/IPC-SysV/actions/runs/138887686

File: t/ipcsysv.t [hunk 1]
```diff
@@ -8,6 +8,10 @@
 #
 ################################################################################
 
+use strict;
+use warnings;
+
+our %Config;
```
Comment (Leont) [inline t/ipcsysv.t:14]: That shouldn't be necessary, right? Because %Config is imported.

Comment (atoomic) [reply t/ipcsysv.t:14]: unfortunately this is required, `%Config` is used in the `BEGIN` block and parsed before

Steps to reproduce
```
> echo 'use strict; use warnings; BEGIN { require Config; import Config; print $Config{d_sem} }' > test.pl
> perl5.30.1 test.pl
Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at test.pl line 1.
BEGIN not safe after errors--compilation aborted at test.pl line 1
```

Comment (atoomic) [reply t/ipcsysv.t:14]: for readability, you can also reproduce the issue with that simple script
```perl
use strict;
use warnings;

BEGIN {
  require Config; import Config;
  $Config{'d_sem'} and print q[ok\n];
}
```

Comment (Leont) [reply t/ipcsysv.t:14]: Or you just say `use Config;` before the begin block?

Comment (atoomic) [reply t/ipcsysv.t:14]: sure but was trying to provide the minimal patch for adding strict/warnings and was avoiding refactor when unnecessary

**Comment by mhx** [comment]
Did you test this (and the XSLoader change) with older Perl versions?

**Comment by mhx** [comment]
Found a 5.8.9 binary on another machine and this seems fine.

**Comment by Leont** [comment]
XSLoader is 5.6+, so that isn't a concern :-)

**Comment by atoomic** [comment]
also note that this is now smoked using different perl versions :-)
https://github.com/mhx/IPC-SysV/runs/798998515

**Comment by mhx** [comment]
> also note that this is now smoked using different perl versions :-)
> https://github.com/mhx/IPC-SysV/runs/798998515

Cool! :)

**Pull request merged**